sequelize-app/README.md

```diff
@@ -0,0 +1,22 @@
+# Build and run
+Install depedencies
+```
+npm install
+```
+
+Open config/config.json and update DB settings for "development"
+
+```
+"development": {
+    "username": <username>,
+    "password": <password>,
+    "database": <database>,
+    "host": <host>,
+    "dialect": "postgres"
+  }
+```
+
+Run via the following command
+```
+DEBUG=sequelize-app:* npm start
+```
```

Comment:
Can we say:
- We are using express for handling REST Api's.
- Which files are boilerplate for express.
- Which are the sequelize ORM files